modules/stripcontrols.cpp

```diff
@@ -20,35 +20,68 @@ class CStripControlsMod : public CModule {
   public:
     MODCONSTRUCTOR(CStripControlsMod) {}
 
-    EModRet OnPrivCTCP(CNick& Nick, CString& sMessage) override {
-        sMessage.StripControls();
+    EModRet OnChanCTCPMessage(CCTCPMessage& Message) override {
+        CString sText = Message.GetText();
```

Comment:
Message.SetText(Message.GetText().StripControls_n());

would be simpler

Reply:
Fixed and force pushed.